easypost/requestor.py

```diff
@@ -117,9 +117,10 @@ def requests_request(self, method, abs_url, headers, params):
             http_body = result.text
             http_status = result.status_code
         except Exception as e:
+            breakpoint()
```

Comment:
Please double check your own code prior to pushing it (self-review each file before you commit in your IDE), stuff like this should never make it into the repo.